Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -304,16 +304,16 @@ void CodeViewWidget::Update(const Core::CPUThreadGuard* guard)
   for (int i = 0; i < rowCount(); i++)
   {
     const u32 addr = AddressForRow(i);
-    const u32 color = PowerPC::debug_interface.GetColor(guard, addr);
+    const u32 color = m_system.GetPowerPC().GetDebugInterface().GetColor(guard, addr);
```

Comment:
You should probably save the debug interface in a variable here. There's a lot of calls to `GetDebugInterface` here and they're all in a loop.

Reply:
Yeah, I was a bit sloppy on the Qt files because I expect to rework them again once the global instance no longer exists. But you're probably right, I updated them.